docs/setup-and-installing.md

```diff
@@ -132,6 +135,7 @@ Build the Unreal GDK module dependencies which the Starter Project needs to work
 1. On the Unreal Editor toolbar, open the **Play** drop-down menu.
 1. Under **Multiplayer Options**, enter the number of players as **2** and check the box next to Run **Dedicated Server**. Then, under Modes, select **New Editor Window (PIE)**.
 1. On the toolbar, click **Play** to run the game.
+1. When you're done, click **Stop** to stop the client, and click the **Stop** button in the SpatialOS secion of the toolbar to stop your local SpatialOS deployment.
```

Comment:
Should this be "client-worker", or is it just a standard game client here?

Reply:
I think either is fine, the instructions would work for both. the only thing you don't have to do in native unreal is launching Spatial

Reply:
(notes from a previous chat about this)

Full name Client Worker, but Client is accepted.
Full name Server Worker, but Server is accepted.

The term worker will be used in technical docs, while just client and server suffice in marketing and high level docs.

This is technical documentation so imma change it.

Reply:
Thanks @oblm! 

I think @ElleEss is on a mission to push for us to use "client-worker" and "server-worker" (in full, and hyphenated) at all times... we should check with the main stakeholders from that original doc though, and then add it to the style guide.